storm/templates/supervisor-deployment.yaml

```diff
@@ -54,3 +55,4 @@ spec:
                 secretKeyRef:
                   name: "{{ .Release.Name }}-{{ .Values.thresh.secretSuffix }}"
                   key: password
+{{- end}}
```

Comment:
what about the pvcs and services. Also should make it possible to disable the monasca-thresh-init job in the monasca helm chart